pkg/db/models/prow.go

```diff
@@ -175,3 +176,16 @@ type ProwPullRequest struct {
 	// MergedAt contains the time retrieved from GitHub that this PR was merged.
 	MergedAt *time.Time `json:"merged_at,omitempty" gorm:"merged_at"`
 }
+
+type JobVariants struct {
+	Release               string
+	FromRelease           string
+	Platform              string
+	Architecture          string
+	Network               string
+	Topology              string
+	NetworkStack          string
+	CloudRegion           string
+	CloudZone             string
+	ClusterVersionHistory []string
+}
```

Comment:
Just a thought, but I wonder if we shouldn't just actually store this information for a job run instead of throwing it away.  

We could address that once we get cluster-data.json written out and consumed by sippy

Reply:
So this is getting pulled in on import now and feeding the variants.  We can look to build that out more for things like CloudRegion, zone & ClusterVersionHistory